hexnumber/hexnumber.py

```diff
@@ -0,0 +1,84 @@
+import sys
```

Comment:
Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 added hexnumber and test 
 Mistakes: 
 Where have you found such homework(added)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(hexnumber), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do(and)? It is not from ['Added', 'Deleted'].